lib/wash/streamer.rb

```diff
@@ -17,8 +17,8 @@ def write(chunk)
           puts("200")
           @first_chunk = false
         end
-        STDOUT.print(chunk)
-        STDOUT.flush
+        $stdout.print(chunk)
```

Comment:
Why this change?

Reply:
See the note on https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/output-matcher.

Reply:
Ah ok, makes sense. Thanks.